tests/test_dicom/test_dicom_anonymise.py

```diff
@@ -1,7 +1,84 @@
-from pprint import pprint
-from pydicom import datadict
-from pymedphys.dicom import anonymise_dicom_dataset, IDENTIFYING_KEYWORDS
-from pymedphys.dicom import BaselineDicomDictionary
+
```

Comment:
@SimonBiggs,
 Please note the following tests. Happy to add to these soon.

This PR is getting big - shall we aim to merge this in once you're happy and cover the actual interface in another PR?

If you think more testing is required, I understand. Just want to see if we're on the same page on PR scope.